buildlib/pr/codestyle.yml

```diff
@@ -63,6 +63,33 @@ jobs:
           fi
         condition: eq(variables['Build.Reason'], 'PullRequest')
 
+  - job: author
+    displayName: AUTHORS file check
+    pool:
+      name: MLNX
+      demands:
+      - ucx_docker -equals yes
+    container: fedora
+    steps:
+      - checkout: self
+        clean: true
+        fetchDepth: 100
+        retryCountOnTaskFailure: 5
+
+      - bash: |
+          set -eE
+          set -x
+          source ./buildlib/az-helpers.sh
```

Comment:
why here and not inside the script? also, which function of `az-helpers.sh` are you using?

Reply:
removed